<?xml version="1.0" ?>

<launch>    
    <param name="gazebo"                   value="false"    type="bool"/>
    <param name="gazebo_robot_name"        value="thormang3"/>
    
    <param name="offset_table"             value="$(find thormang3_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find thormang3_manager)/config/THORMANG3.robot"/>
    <param name="init_file_path"           value="$(find thormang3_manager)/config/dxl_init.yaml"/>
    <param name ="action_file_path"        value="$(find thormang3_action_module)/data/motion_4096.bin"/>
        	 
    <!-- Action Editor -->
    <node name="thormang3_action_editor" pkg="thormang3_action_editor" type="thormang3_action_editor" output="screen"/>
</launch>
